Extensions/MGLMapView+Example.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import Mapbox
+import Turf
+
+extension MGLMapView {
+    func coordinateBoundsInset(_ inset: CGSize) -> MGLCoordinateBounds {
+        return convert(bounds.insetBy(dx: inset.width, dy: inset.height), toCoordinateBoundsFrom: nil)
+    }
+}
+
+
+extension MGLStyle {
+    func addDebugLineLayer(identifier: String, coordinates: [CLLocationCoordinate2D], color: UIColor = UIColor.purple) {
```

Comment:
Ultimately, we’ll need to move these extensions from the example application to MapboxNavigation so that developers don’t have to copy the whole file.

Reply:
Done.